src/main/java/org/cyclops/capabilityproxy/block/BlockEntityCapabilityProxy.java

```diff
@@ -0,0 +1,55 @@
+package org.cyclops.capabilityproxy.block;
+
+import com.google.common.collect.Lists;
+import net.minecraft.block.SoundType;
+import net.minecraft.block.material.Material;
+import net.minecraft.block.properties.PropertyDirection;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.util.EnumFacing;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+import org.cyclops.capabilityproxy.tileentity.TileEntityCapabilityProxy;
+import org.cyclops.cyclopscore.block.property.BlockProperty;
+import org.cyclops.cyclopscore.config.configurable.ConfigurableBlockContainer;
+import org.cyclops.cyclopscore.config.extendedconfig.ExtendedConfig;
+
+/**
+ * This block will forward capabilities from an entity at the target side to all sides.
+ * @author josephcsible
+ */
+public class BlockEntityCapabilityProxy extends ConfigurableBlockContainer {
+
+    @BlockProperty
+    public static final PropertyDirection FACING = PropertyDirection.create("facing", Lists.newArrayList(EnumFacing.VALUES));
+
+    private static BlockEntityCapabilityProxy _instance = null;
+
+    /**
+     * Get the unique instance.
+     * @return The instance.
+     */
+    public static BlockEntityCapabilityProxy getInstance() {
+        return _instance;
+    }
+
+    /**
+     * Make a new block instance.
+     * @param eConfig Config for this block.
+     */
+    public BlockEntityCapabilityProxy(ExtendedConfig eConfig) {
+        super(eConfig, Material.GROUND, TileEntityCapabilityProxy.class);
+
+        setHardness(2.0F);
+        setSoundType(SoundType.STONE);
+    }
+
```

Comment:
The reason I didn't implement that in this PR is that I'm not sure how to make it not conflict with the filtering I eventually want (which IMO is more important).

Reply:
I would then either implement `onBlockActivated` without the concept of filtering, implement/design such a filtering mechanic so that we know how to work around this.

Reply:
My particular hang-up is that I wanted right-clicking to open the filter interface. I just thought of an idea, though: on the target side, right-clicking opens the filter; on the other 5 sides, it forwards it.

Reply:
Makes sense to do it like that.

Some alternatives:
* Filters must be set when the proxy is in item-mode, right clicking in the air could open a gui to configure it.
* Introduce a new item with which you can right click on an entity proxy to configure its filter.

Not sure which one might be better, they all have their pros and cons. But that's not something we have to decide now in any case.